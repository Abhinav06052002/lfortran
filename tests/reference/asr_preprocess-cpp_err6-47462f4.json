{
    "basename": "asr_preprocess-cpp_err6-47462f4",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/cpp_err6.f90",
    "infile_hash": "ac85436de6965d6e21140cdd72302a260de0f86538d86bd9b064a919",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr_preprocess-cpp_err6-47462f4.stderr",
    "stderr_hash": "57201cc657b2a6d7331f83df50256cb0e491df637b57695c86f546a2",
    "returncode": 2
}