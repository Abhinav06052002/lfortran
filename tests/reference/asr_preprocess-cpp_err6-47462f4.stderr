C preprocessor error: Unexpected token type '(', expected type ')'
 --> tests/errors/cpp_err6.f90:2:13
  |
2 | #if 1 + (1+2 (
  |             ^^^ 
